command/certificate/fingerprint.go

```diff
@@ -40,9 +71,25 @@ func fingerprintAction(ctx *cli.Context) error {
 		return err
 	}
 
-	crt, err := pemutil.ReadCertificate(ctx.Args().First())
-	if err != nil {
-		return err
+	var (
+		crt      *x509.Certificate
+		err      error
+		roots    = ctx.String("roots")
+		insecure = ctx.Bool("insecure")
+		crtFile  = ctx.Args().First()
+	)
+
+	if strings.HasPrefix(crtFile, "https://") {
```

Comment:
It doesn't have to be `https://, we can should accept things like `tcp://` or `tls://`

We should do the same in `step certificate inspect`, use a method for both.

Reply:
Add docs for the prefixes supported.